Spec/RestClient.swift

```diff
@@ -832,6 +832,24 @@ class RestClient: QuickSpec {
                 let object = try! NSJSONSerialization.JSONObjectWithData(transport.rawDataSent.first!, options: NSJSONReadingOptions(rawValue: 0))
                 expect(NSJSONSerialization.isValidJSONObject(object)).to(beTrue())
             }
+
+            // RSC7a
+            it("X-Ably-Version must be included in all REST requests") {
+                let options = AblyTests.commonAppSetup()
+                let client = ARTRest(options: options)
+                client.httpExecutor = testHTTPExecutor
+                waitUntil(timeout: testTimeout) { done in
+                    client.channels.get("test").publish(nil, data: "message") { error in
+                        expect(error).to(beNil())
+                        guard let headerAblyVersion = testHTTPExecutor.requests.first?.allHTTPHeaderFields?["X-Ably-Version"] else {
+                            fail("X-Ably-Version header not found"); done()
+                            return
+                        }
+                        expect(headerAblyVersion) == ARTDefault.version()
```

Comment:
Whilst I appreciate this test is accurate, I think it would be useful to in fact be a more explicit and expect 0.8 for example in the string.  I appreciate when we upgrade to 0.9 it will break, but personally I like that as the test expectation has broken.  For example, if someone mistakenly upgrade to 0.99, this test would continue to pass